### PR TITLE
dq comparions and dq validation testable

### PR DESCRIPTION
**Pull request by dramala**
## Summary

Add new test tab to Relation Validation and Relation comparison elements

## How did you test this change?

- [ ] Test(s) added
- [x] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

File: packages/legend-extension-dsl-data-quality/src/components/states/DataQualityTestableState.ts
Comment (yash0024) [inline packages/legend-extension-dsl-data-quality/src/components/states/DataQualityTestableState.ts:1]: Are you able to move `resolveRuntimesFromQuery` (and other shared code with the Function test runner) to TestableUtils.ts? Not particularly important, up to you , can be a follow up MR :)

File: packages/legend-extension-dsl-data-quality/src/components/DataQualityTestsEditor.tsx
Comment (yash0024) [inline packages/legend-extension-dsl-data-quality/src/components/DataQualityTestsEditor.tsx:1]: Nit: CSS leaks like `testable-test-explorer__item` -- can these be moved to legend-art?

File: packages/legend-application-studio/src/stores/editor/editor-state/element-editor-state/testable/TestAssertionState.ts [hunk 1]
```diff
@@ -326,6 +329,38 @@ export class EqualToJsonAssertionState extends TestAssertionState {
...
+ * entirely blank are dropped so an empty expected relation
+ * stays empty after generating.
+ */
+const parseRelationTableString = (
+  text: string,
+): { columns: string[]; rows: string[][] } => {
+  const lines = text.split('\n');
+  if (!lines.length) {
+    return { columns: [], rows: [] };
+  }
```
Comment (yash0024) [inline packages/legend-application-studio/src/stores/editor/editor-state/element-editor-state/testable/TestAssertionState.ts:344]: Can we add a few unit tests for this? Maybe this doesn't work well if the row values have | themselves, but that is likely an issue on engine.... But can you add tests to check for empty row values etc.?

**Comment by yash0024** [comment]
Let's add a roundtrip test with testSuites
